code/modules/events/crystal_event.dm

```diff
@@ -109,14 +109,26 @@ This section is for the event controller
 	for(var/i in 1 to 6)
 		spawn_anomaly(spawners)
 
-	addtimer(CALLBACK(src, .proc/more_portals, GLOB.crystal_invasion_waves[wave_name]), 15 MINUTES)
+	var/list/crystal_spawner_turfs = list()
+	for(var/turf/around_turf in range(6, dest_crystal.loc))
```

Comment:
```suggestion
	for(var/t in RANGE_TURFS(6, dest_crystal.loc))
		var/turf/around_turf = t
```

Reply:
oh? there is something like that??? great!